lib/backends/smartos/tasks/machine_migrate.js

```diff
@@ -803,6 +803,76 @@ function setAutoboot(callback) {
 }
 
 
+function setIndestructibleZoneroot(callback) {
```

Comment:
Why this function signature if this callback is never called?

Reply:
Copied from other examples I believe, but yes I've noticed that callback wasn't used and it bugged me - a good time to stop it :)